hack/runtime-migrator/README-rr.md

```diff
@@ -1 +1,174 @@
-# Runtime Restorer
\ No newline at end of file
+# Runtime Restore
+
+The `runtime-restore` application
+1. connects to a Gardener project, and KCP cluster
+2. retrieves all existing shoot specifications
+3. for each runtime on input list:
+   1. gets shoot, Cluster Role Bindings and OpenIDConnect resources from the backup
+   2. patches shoot if needed 
```

Comment:
Do I understand correctly that it applies `Runtime CR` that results in patching the shoot if needed?

Reply:
No, it patches the shoot with shoot spec from backup. I will update this description to clarify what actually happens.